serialfile.go

```diff
@@ -176,3 +187,17 @@ func (f *serialFile) Size() (int64, error) {
 
 var _ Directory = &serialFile{}
 var _ DirIterator = &serialIterator{}
+
+func filterFiles(filter *Filter, files []os.FileInfo) (res []os.FileInfo) {
+	for _, file := range files {
+		name := file.Name()
+		if file.IsDir() {
+			name = fmt.Sprintf("%s/", name)
```

Comment:
What's this doing?

Reply:
Nothing, it was left over from when ShouldExclude accept path name.